kolibri/core/discovery/test/test_network_search.py

```diff
@@ -151,13 +151,6 @@ def test_irreconcilable_naming_conflict(self):
         for service in services:
             service.unregister()
 
-    def test_excluding_local(self):
-        initialize_zeroconf_listener()
-        register_zeroconf_service(MOCK_PORT, MOCK_ID)
-        assert len(get_available_instances()) == 1
-        assert len(get_available_instances(include_local=False)) == 0
-        unregister_zeroconf_service()
-
     def tearDown(self):
         unregister_zeroconf_service()
```

Comment:
If you are getting contamination between tests, then you might need to call an explicit:
`call_command("flush", interactive=False)`
in the `tearDown` method because of the `TransactionTestCase`.

Reply:
I took this functionality out of the function being tested ... I'd planned on writing some tests for the caching function, but didn't get to it yet. So right now I don't think `TransactionTestCase` is even being used!